lib/elastic_apm/spies/s3.rb

```diff
@@ -87,8 +87,17 @@ def self.prepended(mod)
 
               resource = "#{SUBTYPE}/#{bucket_name || 'unknown-bucket'}"
               context = ElasticAPM::Span::Context.new(
+                db: {
+                  instance: config.region,
+                  type: SUBTYPE
+                },
```

Comment:
Not strictly a _db_. Is this per the spec?

Reply:
Yes, see [here](https://github.com/elastic/apm/blob/master/specs/agents/tracing-instrumentation-db.md#aws-s3)